locallibrary/src/services/author.service.ts

```diff
@@ -14,4 +14,12 @@ export const getAllAuthorsWithBooks = async () => {
   });
   
   return authors;
+};
+
+export const getAuthorById = async (id: number) => {
+  const author = await authorRepository.findOne({
+      where: { id },
+      relations: ['books'],  
+  });
+  return author;
```

Comment:
refactor code xíu hè, sửa tương tự nhé
```suggestion
  return await authorRepository.findOne({
      where: { id },
      relations: ['books'],  
  });
```